app/controllers/remote_search_controller.rb

```diff
@@ -1,8 +1,14 @@
 class RemoteSearchController < ApplicationController
   def show
+    @form = SearchForm.new(params[:search_form])
```

Comment:
Is this necessary? The two forms are completely different paths per your implementation (thus far), so maintaining the state for both of them doesn't make any sense from an end-user functionality perspective.